### DISCUSSION TITLE
Control by switch with thermostat over climate

### DISCUSSION THREAD

**Discussion by sltelitsyn** [jmcollin78/versatile_thermostat#475] [Ideas]
Hello.
Thank you for such a nice integration. It works very smoothly. 

I would like to ask, whether my next suggestion is possible?

I have a mobile air conditioner, which in cool mode always spins its fan. 
Is it possible to somehow configure or implement later a switch, which would direct the AC to go to "off" mode instead of spinning indefinitely? In this mode it sucks hot air outside and wastes all efforts to cool.
 I'd like to save a possibility to change the temperature by the integration, but in case of reaching a desired temperature, just shut off the AC.

Best regards,
Sergei.

**Comment by jmcollin78** [comment]
Hello,

As designed, it is the user responsability to turn on or off the device. Once turned on, VTherm ensure regulation.
This is because, if I auto turn off, you will ask to auto turn on when temperature is far from target.

And I (and you I guess) don't want that VTherm turns on your heaters when you don't need it (because you are on holidays maybe).

So for now, I don't to change this.

BUT, many people and I, need something like that to minimize the power consumption with AC system that always consume (for fan) when they are on, event if the temperature is good.

If you have some suggestions, it could be interesting to discuss about that.

The point is not how to shut off but more how/when to turn on.

**Reply by sltelitsyn** [reply]
Hello. 

Thank you for your response. 

One of my suggestions would be a separate function like with windows, which would require a presence sensor (like a device tracker) for verification of presence at home and optional schedule to generally permit or prohibit autolaunch. 

For my automations I do that with device tracker and planned to automate versatile thermostat the same way. 

Thank you!

**Reply by jmcollin78** [reply]
Another suggestion should be :
1. turn-off if option is selected and target - current > threshold,
2. notify to user to turn-on (to ask for a user confirmation) when needed to turn on.

**Reply by Mels-V** [reply]
Hello @jmcollin78 

I’m running a similar setup myself. I have an air conditioner with a fan that always stays on. To manage this, I’m using the Smart Dual Thermostat integration, which allows me to create a virtual thermostat with two separate outputs—one for cooling and one for heating. Each output gets activated if the current temperature differs by +/- 0.3 degrees from the target temperature. This prevents the unit from switching on and off too frequently.

I’ve linked these outputs to two separate input booleans, which are then tied to automations. The logic is as follows:

	•	If the heating boolean is active, the air conditioner is set to 30°C in heating mode.
	•	If the cooling boolean is active, the air conditioner switches to 18°C in cooling mode.
	•	Lastly, I have an automation that turns off the unit when both booleans are low.

This setup essentially uses the air conditioner as a heating or cooling actuator. Even if the unit turns off, it will automatically turn on again, and I control the on/off behavior by toggling the thermostat. Eventually, I integrated this working principle into VTherm by using a thermostat instead of climate control.

Hope this helps!
Mels
